etc/webstorm.profile

```diff
@@ -0,0 +1,36 @@
+# Firejail profile for WebStorm
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/webstorm.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+noblacklist ${HOME}/.WebStorm*
+noblacklist ${HOME}/.gitconfig
+noblacklist ${HOME}/.jack-server
+noblacklist ${HOME}/.jack-settings
+noblacklist ${HOME}/.java
+noblacklist ${HOME}/.local/share/JetBrains
+noblacklist ${HOME}/.ssh
+noblacklist ${HOME}/.tooling
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-passwdmgr.inc
```

Comment:
Webstorm is for Javascript development, right? We can probably also have `include /etc/firejail/disable-devel.inc` and possibly `include /etc/firejail/disable-interpreters.inc`.